consensus/src/ledger.rs

```diff
@@ -216,6 +216,7 @@ impl OperationLedgerInterface for Operation {
             }
             // roll sale is handled separately with a delay
             models::OperationType::RollSell { .. } => {}
+            models::OperationType::ExecuteSC { .. } => {}
```

Comment:
max_gas * gas price should be credited to the block creator at that point

Reply:
Is this somewhere in the [spec](https://github.com/massalabs/massa/blob/ebeca6cf9c515ed62b88a143f769c883d8ebfdb8/docs/execution.md)?

Reply:
> ExecuteSC spends max_gas * gas_price + coins from the sender address in the CSS ledger, which must therefore be available

That's not so clear but from what I understand here max_gas * gas_price is transferred from the emitter to the block creator at the same time that it would have been if it was just a transaction.

Reply:
I asked the question on the specs PR #1788

Reply:
I think it is pretty well specified:

"ExecuteSC spends max_gas * gas_price + coins from the sender address in the CSS ledger"
...
"credit the block creator in the SCE ledger with `op.max_gas * op.gas_price`"